### DISCUSSION TITLE
请问protable的列控制显示隐藏功能，怎么设置默认选中呢？

### DISCUSSION THREAD

**Discussion by githubqh** [ant-design/pro-components#5440] [Q&A]
![image](https://user-images.githubusercontent.com/105548468/176078421-ee835b27-d893-4849-b251-6fb27cf9502c.png)
![image](https://user-images.githubusercontent.com/105548468/176078479-b1f16ae8-c75e-44fe-a71a-3f57a3a36909.png)

**Comment by PlutoCA** [comment]
https://procomponents.ant.design/components/table#columnsstatetype
